examples/audio plugin host/CMakeLists.txt

```diff
@@ -27,8 +27,7 @@ include(Reprojucer)
 
 set(JUCE_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../JUCE" CACHE PATH "Path to JUCE")
 
-set(Plugin_Host_jucer_FILE
-  "${JUCE_ROOT}/examples/audio plugin host/Plugin Host.jucer"
+set(Plugin_Host_jucer_FILE "${JUCE_ROOT}/examples/audio plugin host/Plugin Host.jucer"
```

Comment:
Should you consider using an automatic linter to prevent this kind of Style diffs ? I opened an issue about it https://github.com/McMartin/JUCE.cmake/issues/29